source/views/contacts/contact-list.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @flow
+ * All About Olaf
+ * iOS Contact page
+ */
+
+import React from 'react'
+import {SectionList, StyleSheet} from 'react-native'
+import {ListSeparator, ListSectionHeader} from '../components/list'
+import {ListEmpty, ListFooter} from '../components/list'
+import {ContactRow} from './contact-row'
+import {data} from './data'
+import groupBy from 'lodash/groupBy'
+import toPairs from 'lodash/toPairs'
+import * as c from '../components/colors'
+import type {ContactType} from './types'
+
+const AAO_URL = 'https://github.com/StoDevX/AAO-React-Native/issues/new'
+
+const groupContacts = (contacts: ContactType[]) => {
+  const grouped = groupBy(contacts, b => b.category || 'Other')
+  return toPairs(grouped).map(([key, value]) => ({title: key, data: value}))
+}
+
+const styles = StyleSheet.create({
+  listContainer: {
+    backgroundColor: c.white,
+  },
+})
+
+export default class ContactsListView extends React.PureComponent {
+  static navigationOptions = {
+    title: 'Important Contacts',
+    headerBackTitle: 'Contacts',
+  }
+
+  onPressContact = (contact: ContactType) => {
+    this.props.navigation.navigate('ContactsDetailView', {
+      contact,
+    })
+  }
+
+  renderSeparator = () => <ListSeparator />
+
+  renderSectionHeader = ({section: {title}}: any) =>
+    <ListSectionHeader title={title} spacing={{left: 10}} />
+
+  renderFooter = () =>
```

Comment:
Please extract this into a constant outside of the class - it doesn't need to be called as a function. 

Just assign the ListFooter to a constant called "footer" or something. It should be in this file, as it's similar in intent to the styles object.